docs/source/repair/index.rst

```diff
@@ -51,20 +50,43 @@ In a benchmark we ran 9 Scylla 2020.1 nodes on AWS i3.2xlarge machines under 50%
 
 By default Scylla Manager runs repairs with full parallelism, you can change that using :ref:`sctool repair --parallel flag <sctool-repair>`.
 
+Maximal effective parallelism
+=============================
+
+Max parallelism is limited by the ``each node can take part in at most one Scylla repair job at any given moment`` constraint.
+For example, picture a cluster with 2 datacenters, 5 nodes each.
+Max parallelism of repairing keyspace ``my_keyspace with replication = {'class': 'NetworkTopologyStrategy', 'dc1': 2, 'dc2': 3}``
+is equal to 1. That's because each Scylla repair job targets full replica set of repaired token range.
+Since every replica set consists of 2 nodes from ``dc1`` and 3 nodes from ``dc2``, it's not possible to schedule 2 repair jobs
+at the same time (``dc2`` lacks 1 node for it to be possible).
+Repair is performed table by table and keyspace by keyspace,
+so the max effective parallelism might change depending on which keyspace is currently being repaired.
+
 Repair intensity
 ================
 
 Intensity specifies how many token ranges can be repaired in a Scylla node at every given time.
 The default intensity is one, you can change that using :ref:`sctool repair --intensity flag <sctool-repair>`.
 
 Scylla Manager 2.2 adds support for intensity value zero.
-In that case the number of token ranges is calculated based on node memory and adjusted to the Scylla maximal number of ranges that can be repaired in parallel (see ``max_repair_ranges_in_parallel`` in Scylla logs).
+In that case the number of token ranges is calculated based on node memory and adjusted to the Scylla maximal number of ranges that can be repaired in parallel.
 If you want to repair faster, try using intensity zero.
 
 Note that the less the cluster is loaded the more it makes sense to increase intensity.
 If you increase intensity on a loaded cluster it may not give speed benefits since cluster have no resources to process more repairs.
 In our experiments in a 50% loaded cluster increasing intensity from 1 to 2 gives about 10-20% boost and increasing it further will have little impact.
 
+Maximal effective intensity
```

Comment:
How about rewriting as follows (the empty lines are important - if not added, everything will melt into one paragraph after publication).

> Max intensity is calculated based on the ``max_repair_ranges_in_parallel`` value (present in ScyllaDB logs). This value might be different for each node in the cluster.
> 
> The overall max intensity is equal to the maximum ``max_repair_ranges_in_parallel`` value of all repaired nodes.
> 
> However, because each ScyllaDB repair job targets some subset of all nodes and ScyllaDB Manager avoids repairing more than ``max_repair_ranges_in_parallel`` on any node, the max effective intensity for a given repair job is equal to the **minimum** ``max_repair_ranges_in_parallel`` value of nodes taking part in the job.

Reply:
Is there any value in saying: "The overall max intensity is equal to the maximum max_repair_ranges_in_parallel value of all repaired nodes"?
I guess there is, so I've left it, but if the information is not useful to the user, let's skip it.

Reply:
I think it can be skipped.